index.html

```diff
@@ -0,0 +1,199 @@
+              </div>
+            </div>
+          </div>
+          <div class="col-11 col-md-2 d-flex flex-md-column gap-md-1 justify-content-md-start align-items-center justify-content-between project-sec mb-1 me-md-1 m-md-0">
+            <div class="row align-items-center p-2">
+              <div class="col-2 col-md-12 d-md-flex justify-content-md-center p-1">
+                <img src="img/goal.png" class="program-icon">
+              </div>
+              <div class="col-3 p-md-0 col-md-12 d-flex justify-content-center">
+                <p class="p-md-2 text-center mb-0 project-sec-name">MANIFEST YOUR DREAMS</p>
+              </div>
+              <div class="col-7 col-md-12 pe-md-2 ps-md-2">
+                <p class="mb-0 pt-2 pb-2 pe-md-0 ps-md-0 project-desc">Implementing the strategies to move your business, relationships, and health goals forward makes all the difference. We are giving you access to three incredible leaders who will give you the tools to bring your greatness to life. Be prepared to kick off the Summit of Greatness with so much knowledge that your mind will already be blown.</p>
+              </div>
+            </div>
+          </div>
+          <div class="col-11 col-md-2 d-flex flex-md-column gap-md-1 justify-content-md-start align-items-center justify-content-between project-sec mb-1 me-md-1 m-md-0">
+            <div class="row align-items-center p-2">
+              <div class="col-2 col-md-12 d-md-flex justify-content-md-center p-1">
+                <img src="img/network.png" class="program-icon">
+              </div>
+              <div class="col-3 p-md-0 col-md-12 d-flex justify-content-center">
+                <p class="p-md-2 text-center mb-0 project-sec-name">MEET AMAZING PEOPLE</p>
+              </div>
+              <div class="col-7 col-md-12 pe-md-2 ps-md-2">
+                <p class="mb-0 pt-2 pb-2 pe-md-0 ps-md-0 project-desc">The key to achieving greatness is creating inspiring relationships. The type of relationships that will push you to the next level. We are giving you the opportunity to connect with other greatness seekers to build relationships with like-minded individuals. Join us for an evening social with local food, drinks, music, and more!</p>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="row join-container d-md-none p-3 mt-3 mb-3 justify-content-center">
+          <div class="col-10 join-bg pe-4 ps-4 pt-3 pb-3 d-flex justify-content-center">
+            <p class="mb-0 fw-bold">GET YOUR TICKETS NOW!</p>
+          </div>
+        </div>
+
+        <div class="row p-md-5 see-program">
+          <div class="col d-flex justify-content-center">
+            <p class="border-bottom border-2 border-secondary">SEE THE WHOLE PROGRAM</p>
+          </div>
+        </div>
+      </div>
```

Comment:
This is a section of the page and I suggest you make use of the `h2` tag for the heading of the section(main programs)😉. This also makes your project more semantic👍.

also, kindly ensure you stick to the background details given in the design for this section😉. I understand yours looks very cool and blends well in your project👍😉. but for this milestone, Let's try to stick to the given design as this is a requirement👍.

your project
<img width="946" alt="image" src="https://github.com/K0ppai/first-capstone-project/assets/64643995/64036fb2-12b5-4398-87fa-1fe2fd3f3484">

design
<img width="623" alt="image" src="https://github.com/K0ppai/first-capstone-project/assets/64643995/fb13c0ed-c6a9-4317-a075-963c55f71267">

Kindly ensure you make use of the right background properties and font properties too👍.

Reply:
Hello, code reviewer @Ibizugbe , Can I leave the changes for the background image? Because it says image changes are acceptable. Thank you sir.
![image](https://github.com/K0ppai/first-capstone-project/assets/113283174/e91feb51-452c-43fa-ba6a-e91f2d878bfc)